3-UsingAPIs/Week2/homework/ex6-browserDebug/index.js

```diff
@@ -39,9 +41,10 @@ function renderLaureates(laureates) {
 
 async function fetchAndRender() {
   try {
-    const laureates = getData(
+    const laureate = await getData(
       'http://api.nobelprize.org/2.0/laureates?birthCountry=Netherlands&format=json&csvLang=en'
     );
+    const {laureates} =laureate
```

Comment:
Well done 👍  I hope you learned how to use the JS debugger - this will be very useful in the future